modules/dailymotionBidAdapter.js

```diff
@@ -41,14 +41,13 @@ export const spec = {
 
   /**
    * Determines whether or not the given bid request is valid.
-   * The only mandatory parameters for a bid to be valid are the api_key and position configuration entries.
+   * The only mandatory parameter for a bid to be valid is the API key.
    * Other parameters are optional.
    *
    * @return boolean True if this is a valid bid, and false otherwise.
    */
-  isBidRequestValid: () => {
-    const dmConfig = config.getConfig('dailymotion');
-    return !!dmConfig?.api_key;
+  isBidRequestValid: function (bid) {
+    return (typeof bid.params.apiKey !== 'undefined');
```

Comment:
Shouldn't it be more specific and test for typeof `string`?
```suggestion
    return (typeof bid.params.apiKey === 'string');
```